src/rust/ide/lib/view/src/graph_editor/component/breadcrumbs/breadcrumb.rs

```diff
@@ -0,0 +1,416 @@
+//! This module provides a clickable view for a single breadcrumb.
+
```

Comment:
If we change the width of separators, we can observe they are also not aligned with the text:
    ![image](https://user-images.githubusercontent.com/1623053/89508188-dcbf1800-d7cd-11ea-8dbe-2c62d320a123.png)

Reply:
It seems like a wrong assumption. The text is misaligned because it was wrongly using ICON_SIZE to align the text vertically. It was fixed in the previous comment.